src/main/java/com/khoros/twitterapp/services/TwitterService.java

```diff
@@ -82,17 +82,38 @@ public TwitterService(Twitter twitterFactory) {
 
     public Optional<List<Status>> getHomeTimelineFilteredByKeyword(String keyword) throws TwitterServiceException {
 
+        return getFilteredTimeline("home", keyword);
+    }
+
+    public Optional<List<Status>> getUserTimeline() throws TwitterServiceException {
+
+        return getFilteredTimeline("user", null);
+    }
+
+    public Optional<List<Status>> getFilteredTimeline(String timelineType, String keyword) throws TwitterServiceException {
```

Comment:
instead of adding a timeline type, separate the timelines to different methods.  It will reduce the conditionals and keep your code cleaner